pyproject.toml

```
@@ -150,6 +153,25 @@ version = { attr = "integreat_compass.__version__" }
 [tool.setuptools.packages.find]
 include = ["integreat_compass*"]
 
+[tool.bumpver]
+current_version = "2023.3.0"
+version_pattern = "YYYY.MM.INC0[-TAG]"
+commit_message = """
+Bump version to {new_version}
+"""
```

```suggestion

[skip ci]
"""
```
It' important that version bumps do not trigger the CI workflow, otherwise we run into an indefinite loop where a merge on main triggers a version bump, and the version bump triggers another version bump, and so on...

---

Ah, thank you for the explanation. I didn't realize this was part of the issue

---

Maybe not necessarily part of this issue, but easy to miss later on